### ISSUE TITLE
Newsletter Unsubscribe 

### ISSUE THREAD

**Issue by rrbailey89** [JonnyWong16/plexpy#1910]
### Is your feature request related to a problem?

I request that a feature be implemented to allow unsubscribing from self hosted newsletters.

### What is your feature request?

unsubscribing from self hosted newsletters

**Comment by JonnyWong16** [comment]
This would require users to expose their Tautulli instances publically on the internet. You're better off using an email service like [SendGrid](https://sendgrid.com) or [Mailgun](https://mailgun.com) which will handle unsubscribing for you without exposing Tautulli.

**Comment by rrbailey89** [comment]
> This would require users to expose their Tautulli instances publically on the internet. You're better off using an email service like [SendGrid](https://sendgrid.com) or [Mailgun](https://mailgun.com) which will handle unsubscribing for you without exposing Tautulli.

Briefly looking at MailGun, it has a trial that's good for one month. After that, it's not free. SendGrid may work; however, to your point about exposing Tautulli, I feel like that is a non-issue; it's already shipped with the ability to secure it with a username and password as well as the ability to allow guests to log in with plex.tv credentials.

**Comment by gshlomi** [comment]
+1 for the request, my instance is already exposed to my users usage.
Using an external SMTP service negates the syncing of users between the Plex server and Tautulli

**Comment by mylogon341** [comment]
@rrbailey89 @gshlomi Did either of you figure this out in a graceful way at all? I'm trying to set this up myself. I can easily get it to send SMTP email but want for users to be able to opt out of them without having to awkwardly message me with "stop sending me this please". I've looked at a number of providers such as sendgrid, but they don't seem to able to achieve this without hooking up a custom domain, which will cost money. I'm not bothered enough about this to part way with too much cash at all, but don't mind throwing a few £ (if needs be) at it to get this cool feature off the ground.
Thanks

**Comment by rrbailey89** [comment]
> @rrbailey89 @gshlomi Did either of you figure this out in a graceful way at all? I'm trying to set this up myself. I can easily get it to send SMTP email but want for users to be able to opt out of them without having to awkwardly message me with "stop sending me this please". I've looked at a number of providers such as sendgrid, but they don't seem to able to achieve this without hooking up a custom domain, which will cost money. I'm not bothered enough about this to part way with too much cash at all, but don't mind throwing a few £ (if needs be) at it to get this cool feature off the ground. Thanks

I have not and still say it should be a feature of Tautulli.

![Screenshot_20230930-190359](https://github.com/Tautulli/Tautulli/assets/6540217/66700821-9020-4b8a-b4cf-a538b9354c2c)

**Comment by JonnyWong16** [comment]
You do realize that even if this is a feature of Tautulli you will still probably need a domain pointed to Tautulli, right? Your friends will need some link to click on that goes to your own Tautulli instance.

**Comment by rrbailey89** [comment]
I myself have a domain pointed to Tautulli. I suspect at least 50% of Tautulli users also have domains pointed to there Tautulli instance.

**Comment by JonnyWong16** [comment]
Lol no. It's more likely closer to 5%.

**Comment by trentnbauer** [comment]
+1 for this, I already have tautulli exposed via a Cloudflare tunnel (but with CF auth in front)
Using a reverse proxy we could expose only a subpath (eg 'newsletter'), like mywebsite.com/newsletter
myserver:8181/newsletter could then show a text field for email address and an unsub button
Only proxy'ing the /newsletter will expose that page and nothing else.

**Comment by xenago** [comment]
Yeah this seems like a super useful feature, hopefully it can be implemented!

**Comment by joldnl** [comment]
At least add a feature to control the unsubscribe mail header eg:  `List-Unsubscribe: <mailto: unsubscribe@example.com>` so we can adhere to spamfilters needs/

**Comment by joldnl** [comment]
As this doesn't seem to be implemented any time soon, I am considering building a small laravel app that handles the unsubscribe feature, using the tautulli api to do this.
This way, you can have a unsubscribe link in your mails sent trough smtp of mailgun. I tried using mailgun but they would not accept removing the development setting until I could prove that there is a working unsubscribe form.
